src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -118,39 +114,25 @@
 
 (defn home
   []
-  (let [pending-contact-requests (rf/sub [:activity-center/pending-contact-requests])
-        selected-tab             (or (rf/sub [:messages-home/selected-tab]) :tab/recent)
-        customization-color      (rf/sub [:profile/customization-color])
-        top                      (safe-area/get-top)]
-    [:<>
-     (if (= selected-tab :tab/contacts)
-       [contacts pending-contact-requests top]
-       [chats selected-tab top])
-     [rn/view {:style (style/blur-container top)}
-      (let [{:keys [sheets]} (rf/sub [:bottom-sheet])]
-        [blur/view
-         {:blur-amount   (if platform/ios? 20 10)
-          :blur-type     (if (colors/dark?) :dark (if platform/ios? :light :xlight))
-          :style         style/blur
-          :overlay-color (if (seq sheets)
-                           (theme/theme-value colors/white colors/neutral-95-opa-70)
-                           (when (colors/dark?)
-                             colors/neutral-95-opa-70))}])
-      [common.home/top-nav {:type :grey}]
-      [common.home/title-column
-       {:label               (i18n/label :t/messages)
-        :handler             #(rf/dispatch [:show-bottom-sheet {:content home.sheet/new-chat}])
-        :accessibility-label :new-chat-button
-        :customization-color customization-color}]
-      [quo/discover-card
-       {:banner      (resources/get-image :invite-friends)
-        :title       (i18n/label :t/invite-friends-to-status)
-        :description (i18n/label :t/share-invite-link)}]
-      ^{:key (str "tabs-" selected-tab)}
-      [quo/tabs
-       {:style          style/tabs
-        :size           32
-        :on-change      (fn [tab]
-                          (rf/dispatch [:messages-home/select-tab tab]))
-        :default-active selected-tab
-        :data           (get-tabs-data (pos? (count pending-contact-requests)))}]]]))
+  (let [flat-list-ref (atom nil)]
+    (fn []
+      (let [pending-contact-requests (rf/sub [:activity-center/pending-contact-requests])
+            selected-tab             (or (rf/sub [:messages-home/selected-tab]) :tab/recent)
+            scroll-shared-value      (reanimated/use-shared-value 0)]
+        [:<>
+         (if (= selected-tab :tab/contacts)
+           [contacts
+            {:pending-contact-requests pending-contact-requests
+             :flat-list-ref            flat-list-ref
+             :scroll-shared-value      scroll-shared-value}]
+           [chats
+            {:selected-tab        selected-tab
+             :flat-list-ref       flat-list-ref
+             :scroll-shared-value scroll-shared-value}])
+         [:f> common.home.banner/animated-banner
+          {:content             :chats
+           :flat-list-ref       flat-list-ref
+           :tabs                (get-tabs-data (pos? (count pending-contact-requests)))
+           :selected-tab        selected-tab
+           :on-tab-change-event (fn [tab] (rf/dispatch [:messages-home/select-tab tab]))
```

Comment:
```suggestion
           :on-tab-change-event #(rf/dispatch [:messages-home/select-tab %])
```

Reply:
Actually, it was my first approach, but I was worried the readers didn't know what was the passed argument since this is a custom event handler; for known events handlers (`on-press`, `on-scroll`, etc) I think is ok to pass an anonymous param (`%` or even `%1`, `%2`, `%3`, ...).

So, at the end I decided to give it a name (`tab`), I preferred to be clear  enough. The cost in terms of code length is almost the same.